tests/conftest.py

```diff
@@ -0,0 +1,4 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), 'tests'))) # noqa
```

Comment:
what does this do?

Reply:
It is adding the path to the tests folder to the sys path. Something is wrong with Cameron's setup. He shouldn't need this.